### PR TITLE
Feature/006-Unit-testing -> dev

### PR DESCRIPTION
**Pull request by WMinowski**

### CODE REVIEW

File: UnitTestProject/PostalCodeMaskTest.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+{
+    [TestClass]
+    public class PostalCodeMaskTest
+    {
+        RelationsController relationController = new RelationsController();
+
+        [TestMethod]
+        public void ValueMaskCoincidence()
+        {
+            Assert.AreEqual(relationController.ApplyMask("1488-HH", "NNNN-LL"), "1488-HH");
```
Comment (dmyhun-andersen) [inline UnitTestProject/PostalCodeMaskTest.cs]: Do not write inline.
Use structure AAA:
Arrange
Act
Assert

Also write test cases if (**ApplyMask(string value, string mask)** ) value or mask = String.Empty or value or mask = null.

Comment (dmyhun-andersen) [reply UnitTestProject/PostalCodeMaskTest.cs]: Correct naming:

- TestMaskWithDifferentRegister
- TestMaskWithDifferentCharNumber

and so on

**Comment by dmyhun-andersen** [comment]
Please commit changes with correct displayNames of test cases

**Comment by WMinowski** [comment]
displayNames will be changed in the next feature

**Pull request merged**